### PR TITLE
f

### PR DESCRIPTION
**Pull request by Sargis996**

### CODE REVIEW

**Comment by SargisArzumanyan** [comment]
Overall design is ok, but I have a recommendation to split large chunks of code into smaller more easily understandable parts.
The code works as intended.
The complexity is ok.
Names are ok.
The idea to use correlation matrix heatmap to visualize the relationship between variables is a good one.

**Comment by monica06-yan** [comment]
The code is well-structured and easy to follow.The code correctly splits the dataset into training and testing sets using `train_test_split`.The code trains three different classifiers (Logistic Regression, Naive Bayes, and KNN) and makes good  predictions on the test set.Overall, the code seems well-organized and provides a good balance between clarity and functionality.👏🏽🌟
